src/onboarding/registration/NameAndPicture.tsx

```diff
@@ -50,7 +54,16 @@ function NameAndPicture({ navigation, route }: Props) {
   const asyncKomenciReadiness = useAsyncKomenciReadiness()
   const showGuidedOnboarding = useSelector(showGuidedOnboardingSelector)
   const createAccountCopyTestType = useSelector(createAccountCopyTestTypeSelector)
-  const skipUsername = route.params?.skipUsername //TODO repalce with statsig variable
+  const skipUsername = shouldSkipUsername() //TODO repalce with statsig variable
+  const showAlternatePlaceholder = shouldShowAlternatePlaceholder() //TODO repalce with statsig variable
+  let placeholderUsername = t('fullNamePlaceholder')
+  if (showGuidedOnboarding) {
+    placeholderUsername = t('fullNameOrPseudonymPlaceholder')
+  }
+  if (showAlternatePlaceholder) {
+    placeholderUsername = 'MyCryptoAlterEgo' // not localized
+  }
+
```

Comment:
Small nit: I think this would be cleaner inside of a `getPlaceholderUsername()` function. Similar to how we have a helper function for `goToNextScreen`

Reply:
Then you can avoid using `let` and asigning `placeholderUsername` and just do `return t('fullNameOrPseudonymPlaceholder')` or `return 'MyCryptoAlterEgo'` instead.